src/feed/feed.controller.ts

```
@@ -45,6 +53,17 @@ export class FeedController {
     return this.feedService.getSavedPosts(userId, query.page, query.limit);
   }
 
+  @UseGuards(JwtAuthGuard)
+@Post(':postId/comments')
+async addComment(
+  @Param('postId') postId: string,
```

ESLint

[error] Replace `··` with `····`

(prettier/prettier)